### PR TITLE
Build react dom browser 2

### PR DESCRIPTION
**Pull request by zpao**
Supersedes #4761

- [x] send to build server
- [x] rename to `.min`?
- [x] make sure it's bundled in all the places (eg, bower)
- [ ] update examples (can happen after if we want)

### CODE REVIEW

**Comment by spicyj** [comment]
Can we make a .js and a .min.js for consistency? If people don't look too closely they might not notice that it's three lines.

**Comment by zpao** [comment]
Sure

File: src/React.js [hunk 1]
```diff
@@ -64,4 +64,6 @@ assign(React, {
   ),
 });
 
+React.__sekret_DOM_DO_NOT_USE_OR_YOU_WILL_BE_FIRED = ReactDOM;
```
Comment (cpojer) [inline src/React.js]: wait, I thought this was a joke? In German, Sekret means secretion, so not exactly the word you are looking for here. Use "Geheimnis" instead.

Comment (zpao) [inline src/React.js]: Are you sure it's not the word we're looking for here? (ew)

**Comment by zpao** [comment]
I changed it to `__SECRET_DOM_DO_NOT_USE_OR_YOU_WILL_BE_FIRED` to make our German friends less grossed out. I'm going to do the examples in a separate PR.

**Pull request merged**